dump_utils.cpp

```diff
@@ -147,5 +147,62 @@ bool isHostQuiesced()
     }
     return false;
 }
+
+void createPEL(const std::string& dumpFilePath, const std::string& dumpFileType,
+               const int dumpId, const std::string& pelSev,
+               const std::string& errIntf)
+{
+    try
+    {
+        if (dumpFilePath.empty() || dumpFileType.empty())
+        {
+            log<level::WARNING>(
+                fmt::format("User data is not sufficient to "
+                            "create a PEL message for dump delete/offload")
+                    .c_str());
+            throw std::runtime_error("Dump delete/offload PEL not created due "
+                                     "to insufficient info passed");
+        }
```

Comment:
not required, worst case in the pel these fields will be empty, we are not going to crash if they are empty

Reply:
Agree. Removed